File: Entities/Movie.cs
using Cinema.Entities.Enumerations;
using Microsoft.AspNetCore.Http;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace Cinema.Entities
{
    public class Movie
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public int Length { get; set; }

        [DisplayFormat(DataFormatString = "{0:d}")]
        public DateTime DateOfRelease { get; set; }
        public string TrailerUrl { get; set; }
        public string CoverImageUrl { get; set; }
        public MovieCategory Category { get; set; }

        [DisplayFormat(DataFormatString = "{0:d}")]
        public DateTime PlayingDate { get; set; }

        [DisplayFormat(DataFormatString = "{0:t}")]
        public DateTime PlayingTime { get; set; }

        [NotMapped]
        public IFormFile Image { get; set; }
    }
}

File: client/src/app/_models/movie.ts
export interface Movie {
  id: number;
  title: string;
  description: string;
  length: number;
  dateOfRelease: Date;
  trailerUrl: string;
  coverImageUrl: string;
  category: MovieCategory;
  playingDate: Date;
  playingTime: Date;
  image: string;
}

export enum MovieCategory {
  Horror = 1,
  Science_Fiction,
  Akcja,
  Przygodowy,
  Animowany,
  Komedia,
  Dramat,
  Thriller,
  Western,
  Historyczny
}

